calculator.rb

```diff
@@ -0,0 +1,90 @@
+# Calculator (Dinner) - Loraine Kanervisto
+
+# Starts with user prompts and data collection
+
+#Collects first number
+puts "You find a calculator on the ground. You pick it up. The screen says:\n
+Enter a number>>"
+first_num = gets.chomp.to_f
+
+#Checks first number
+while first_num != "0" && first_num.to_i == 0
```

Comment:
Ah, you're right. I updated the while loop to include a break if the string is "0." Converted both the first_num and second_num to integers instead. Does that seem like a good catch-all for numerical values, or should I be customizing them based on the operator?